android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -79,9 +87,10 @@ public boolean disableExclusiveNfc() {
             } else {
                 disableForegroundDispatch(adapter);
             }
-            return true;
+            return NfcStatus.AVAILABLE;
         }
-        return false;
+        if (handleUnavailableNfc) toast("NFC is not available on this device");
```

Comment:
@Franzaine The function promises to disable NFC. If there is no NFC the promise has still been kept, so there is no need to handle unavailable NFC. No action is needed by the user so no information is required,